src/main/java/life/qbic/io/PetabParser.java

```diff
@@ -0,0 +1,85 @@
+package life.qbic.io;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import life.qbic.model.petab.PetabMetadata;
+
+public class PetabParser {
+
+  private final String META_INFO_YAML = "metaInformation.yaml";
```

Comment:
MAJOR NIT: Could this be provided case insensitive? I forsee that this file will be provides as metainformation.yaml instead of metaInformation.yaml

Reply:
good point, I will compare case insensitive